src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random number = new Random();
```

Comment:
Having two instances of Random is redundant. You can use a single instance to generate all the random numbers needed.